invisible_cities/reco/paolina_functions_test.py

```diff
@@ -209,3 +209,27 @@ def test_voxelize_single_hit():
     hits = [BHit(1, 1, 1, 100)]
     vox_size = np.array([10,10,10], dtype=np.int16)
     assert len(voxelize_hits(hits, vox_size)) == 1
+
+def test_length():
+    voxel_spec = ((10,10,10,  1000),
+                  (10,10,11,     1),
+                  (10,10,12,     2),
+                  (10,10,13,     4),
+                  (10,10,14,     8),
```

Comment:
The 'interesting' energies in `voxel_spec` give the impression that the energies somehow matter in this test. AFAICT, the energies are irrelevant, so don't make them look interesting. It would be best if you could get away with `None` or `'dummy'`. But I suspect that you will need the energies to be

+ numeric
+ greater than zero

Reply:
Yes, they must be number different from zero. What do you suggest?

Reply:
1?

Reply:
Ah, ok, I was thinking more about random numbers, but maybe it's an overkill...

Reply:
The energy values are completely irrelevant for the test, beyond being nonzero. So I would follow the principle of making them as dummy as possible. But nonzero random numbers would be fine too.